### PR TITLE
Feature/935 edit user groups

### PR DESCRIPTION
**Pull request by kelvincy**

### CODE REVIEW

File: frontend/zac-ui/libs/features/zaak-detail/src/lib/activiteiten/activiteiten.service.ts [hunk 1]
```diff
@@ -17,9 +17,9 @@ export class ActiviteitenService {
     return this.http.Get<UserSearch>(endpoint);
   }
 
-  getUserGroups(searchInput: string): Observable<UserGroupSearch>{
+  getUserGroups(searchInput: string): Observable<UserGroupList>{
```
Comment (svenvandescheur) [inline frontend/zac-ui/libs/features/zaak-detail/src/lib/activiteiten/activiteiten.service.ts:20]: Can we add the API documentation as docstrings in services?

File: frontend/zac-ui/libs/features/zaak-detail/src/lib/keten-processen/keten-processen.service.ts [hunk 2]
```diff
@@ -141,9 +140,9 @@ export class KetenProcessenService {
     return this.http.Get<UserSearch>(endpoint);
   }
 
-  getUserGroups(searchInput: string): Observable<UserGroupSearch>{
+  getUserGroups(searchInput: string): Observable<UserGroupList>{
```
Comment (svenvandescheur) [inline frontend/zac-ui/libs/features/zaak-detail/src/lib/keten-processen/keten-processen.service.ts:143]: Can we add the API documentation as docstrings in services?

File: frontend/zac-ui/libs/features/auth-profiles/src/lib/user-groups/create-group/create-group.component.ts [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+  readonly createGroupErrorMessage = "Er is een fout opgetreden bij het aanmaken van de gebruikersgroep."
+  readonly editGroupSuccessMessage = "De gebruikersgroep is gewijzigd."
+  readonly editGroupErrorMessage = "Er is een fout opgetreden bij het wijzigen van de gebruikersgroep."
+
+  constructor(
+    private fService: FeaturesAuthProfilesService,
+    private modalService: ModalService,
+    private snackbarService: SnackbarService,
+    private fb: FormBuilder,
+  ) {
```
Comment (svenvandescheur) [inline frontend/zac-ui/libs/features/auth-profiles/src/lib/user-groups/create-group/create-group.component.ts:42]: can't we use the generic fieldconfigurating along with the generic form?

**Review by svenvandescheur — CHANGES_REQUESTED** [review]

File: frontend/zac-ui/libs/features/auth-profiles/src/lib/user-groups/create-group/create-group.component.ts (hunk 3 above)
Comment (kelvincy) [reply frontend/zac-ui/libs/features/auth-profiles/src/lib/user-groups/create-group/create-group.component.ts:42]: The generic form here is not sufficient

**Pull request merged**